File: MODUL-1/SOAL 9/PRAK109-2410817120017-RIKA-FAULIANA-RAHMI.py
# Mendefinisikan jumlah pasukan dan jumlah pahlawan
jumlah_pasukan = 958730
jumlah_pahlawan = 5

# Menghitung jumlah pasukan yang harus dikalahkan setiap pahlawan
pasukan_per_pahlawan = jumlah_pasukan // jumlah_pahlawan  # Menggunakan integer division

# Menampilkan hasil
print(f"Jumlah pasukan yang dibawa Yu Zhong = {jumlah_pasukan}")
print(f"Jumlah pahlawan = {jumlah_pahlawan}")
print(f"Jumlah pasukan yang harus dikalahkan setiap pahlawan adalah {pasukan_per_pahlawan} pasukan")

File: MODUL-1/SOAL 9/PRAK109-2410817120017-RIKA-FAULIANA-RAHMI.c
#include <stdio.h>

int main() {
    // Mendefinisikan jumlah pasukan dan jumlah pahlawan
    int jumlahPasukan = 958730;
    int jumlahPahlawan = 5;

    // Menghitung jumlah pasukan yang harus dikalahkan setiap pahlawan
    int pasukanPerPahlawan = jumlahPasukan / jumlahPahlawan;

    // Menampilkan hasil sesuai format yang diminta
    printf("Jumlah pasukan yang dibawa Yu Zhong = %d\n", jumlahPasukan);
    printf("Jumlah pahlawan = %d\n", jumlahPahlawan);
    printf("Jumlah pasukan yang harus dikalahkan setiap pahlawan adalah %d pasukan\n", pasukanPerPahlawan);

    return 0;
}
